.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea/
```

Comment:
You should add this in your own global `.gitignore` rules, not in the project rules.